app/core/user/user.service.ts

```diff
@@ -16,86 +16,86 @@ import { Observable } from 'rxjs/Rx';
  * );
```

Comment:
This file seems a little bit messed up with spaces. Please format the code correctly again and delete unnecessary  spaces